docs/developer/webui/webui_frontend_vscode_dev_env.md

```diff
@@ -101,4 +101,4 @@ Finally, we must change the `RUCIO_AUTH_HOST` and `RUCIO_HOST` in the
 to the rucio container. By default, this should be `https://dev-rucio-1:443`.
 
 Test the setup by logging in using the credentials username `ddmlab`/ password
-`secret`/ account `root` under the VO `Default`.
\ No newline at end of file
+`secret`/ account `root` under the VO `Default`.
```

Comment:
This diff looks a bit weird because as it looks identical, but I checked with `git diff --ignore-all-space` and it shows no diff, so it's only a safe whitespace change.